cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -268,15 +268,15 @@ find_package(PQXX REQUIRED)
 include_directories(SYSTEM ${PQXX_INCLUDE_DIRECTORIES})
 list(APPEND TERRIER_LINK_LIBS ${PQXX_LIBRARIES})
 
-# LLVM 6.0+ NOT USED CURRENTLY (Matt)
-#find_package(LLVM 6.0 REQUIRED CONFIG)
-#message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
-#if (${LLVM_PACKAGE_VERSION} VERSION_LESS "6.0")
-#  message(FATAL_ERROR "LLVM 6.0 or newer is required.")
-#endif ()
-#llvm_map_components_to_libnames(LLVM_LIBRARIES core mcjit nativecodegen native)
-#include_directories(SYSTEM ${LLVM_INCLUDE_DIRS})
-#list(APPEND TERRIER_LINK_LIBS ${LLVM_LIBRARIES})
+# LLVM 7.0+
+find_package(LLVM 7.0 REQUIRED CONFIG)
```

Comment:
This fails on macOS because 7.1 is what's offered. If we're compatible with LLVM 7.1, propose getting rid of the 7.0 field on find_package because it does a strict version check.
```
  Could not find a configuration file for package "LLVM" that is compatible
  with requested version "7.0".

  The following configuration files were considered but not accepted:

    /usr/local/Cellar/llvm@7/7.1.0/lib/cmake/llvm/LLVMConfig.cmake, version: 7.1.0
    /usr/local/Cellar/llvm@7/7.1.0/lib/cmake/llvm/llvm-config.cmake, version: unknown
```
The check on the following lines do a minimum check of 7.0. Does that suffice?

Reply:
Remove the version on the this line and let the following check ensure the 7.0 minimum.

8 also works and I'd recommend using if it it's available on the build machines.

Reply:
Should we just try to make 8 the required version now while we're changing it anyway?

Reply:
Yes, but I'm not sure its available on Travis.